data/mods/Aftershock/mutations/mutations.json

```diff
@@ -290,7 +290,8 @@
     "points": 4,
     "description": "You can now breathe the gasses the mi-go thrive in.",
     "starting_trait": false,
-    "category": [ "MIGO" ]
+    "category": [ "MIGO" ],
+    "flags": [ "MIGO_IMMUNE" ]
```

Comment:
I guess I'm confused how this works is the field immunity for this still working off the trait id and the flag is prework for when it starts to fire off the flag versus the trait or have you done some kind of voodoo where any field that has the word migo in it will now be resisted by this flag?

Reply:
I added 
```
    "immunity_data": { "body_part_env_resistance": [ [ "mouth", 15 ] ], "flags": [ "MIGO_IMMUNE" ] },
```
to the Aftershock migo atmosphere itself, the immunity now checks for the flag. It should all still work, the switch to flags is just so that we don't have two parallel systems of trait-based immunity.